tests/foreman/ui/test_syncplan.py:121: AssertionError

Error Message

AssertionError: assert '' == '2019/07/08 05:07'   + 2019/07/08 05:07

Stacktrace

session = <airgun.session.Session object at 0x7f31a4b9b048>

    @tier2
    def test_positive_end_to_end(session):
        """Perform end to end scenario for sync plan component
    
        :id: 39c140a6-ca65-4b6a-a640-4a023a2f0f12
    
        :expectedresults: All CRUD actions for component finished successfully
    
        :CaseLevel: Integration
        """
        plan_name = gen_string('alpha')
        description = gen_string('alpha')
        new_description = gen_string('alpha')
        with session:
            startdate = (
                    session.browser.get_client_datetime() + timedelta(minutes=10))
            # Create new sync plan and check all values in entity that was created
            session.syncplan.create({
                'name': plan_name,
                'interval': SYNC_INTERVAL['day'],
                'description': description,
                'date_time.start_date': startdate.strftime("%Y-%m-%d"),
                'date_time.hours': startdate.strftime('%H'),
                'date_time.minutes': startdate.strftime('%M'),
            })
            assert session.syncplan.search(plan_name)[0]['Name'] == plan_name
            syncplan_values = session.syncplan.read(plan_name)
            assert syncplan_values['details']['name'] == plan_name
            assert syncplan_values['details']['description'] == description
            assert syncplan_values['details']['enabled'] == 'Yes'
            assert syncplan_values['details']['interval'] == SYNC_INTERVAL['day']
            time = syncplan_values['details']['date_time'].rpartition(':')[0]
>           assert time == startdate.strftime("%Y/%m/%d %H:%M")
E           AssertionError: assert '' == '2019/07/08 05:07'
E             + 2019/07/08 05:07

tests/foreman/ui/test_syncplan.py:121: AssertionError

Hello

this is the error I see:

\>           assert time == startdate.strftime("%Y/%m/%d %H:%M")
E           AssertionError: assert 'Jul 9, 2019 8:54' == '2019/07/09 08:54'
E             - Jul 9, 2019 8:54
E             + 2019/07/09 08:54

--- a/tests/foreman/ui/test_syncplan.py
+++ b/tests/foreman/ui/test_syncplan.py
@@ -118,7 +118,7 @@ def test_positive_end_to_end(session):
         assert syncplan_values['details']['enabled'] == 'Yes'
         assert syncplan_values['details']['interval'] == SYNC_INTERVAL['day']
         time = syncplan_values['details']['date_time'].rpartition(':')[0]
-        assert time == startdate.strftime("%Y/%m/%d %H:%M")
+        assert time == startdate.strftime("%b %-d, %Y %H:%M")
         # Update sync plan with new description
         session.syncplan.update(
             plan_name,

